### PR TITLE
Column Groups: Fix cell navigation 

### PR DESCRIPTION
**Pull request by amanmahajan7**

### CODE REVIEW

File: src/utils/selectedCellUtils.ts [hunk 1]
```diff
@@ -119,12 +137,58 @@ export function getNextSelectedCellPosition<R, SR>({
     }
   };
 
+  const getParentRowIdx = (parent: CalculatedColumnParent<R, SR>) => {
+    return parent.level + mainHeaderRowIdx;
+  };
+
+  const setHeaderGroupColAndRowSpan = (moveRight: boolean) => {
+    if (moveRight) {
```
Comment (amanmahajan7) [inline src/utils/selectedCellUtils.ts:145]: Needs special handling for these right and top navigation

**Review by amanmahajan7 — COMMENTED** [review]
There may be some edge cases but this should cover basic navigation

File: src/utils/selectedCellUtils.ts [hunk 2]
```diff
@@ -119,12 +137,58 @@ export function getNextSelectedCellPosition<R, SR>({
     }
   };
 
+  const getParentRowIdx = (parent: CalculatedColumnParent<R, SR>) => {
+    return parent.level + mainHeaderRowIdx;
+  };
+
+  const setHeaderGroupColAndRowSpan = (moveRight: boolean) => {
```
Comment (nstepien) [inline src/utils/selectedCellUtils.ts:144]: Is `moveRight` always about moving right? What about right-to-left layouts? Should this be `moveNext` instead?

**Review by nstepien — APPROVED** [review]

File: src/utils/selectedCellUtils.ts (hunk 2 above)
Comment (amanmahajan7) [reply src/utils/selectedCellUtils.ts:144]: good point

**Review by nstepien — APPROVED** [review]

**Pull request merged**